src/nixfeature.cc

```diff
@@ -30,8 +30,9 @@ namespace nixfeature {
         nix::Feature currFeat = input.entity<nix::Feature>(1);
         //TODO properly implement link type
         struct_builder sb({ 1 }, { "linkType" });
-        sb.set("linkType");
+	std::string s = "linkType";
+        sb.set(s);
```

Comment:
Do you have a (very) recent NIX installed on the system? Because this will if `nix::Value`'s char constructors are not marked explicit.